soc/arm/st_stm32/stm32f4/dts_fixup.h

```diff
@@ -271,30 +271,6 @@
 #define DT_I2S_6_DMA_FEATURES_RX		\
 			DT_ST_STM32_I2S_40015400_RX_DMAS_FEATURES
 
-#define DT_FLASH_DEV_NAME		DT_INST_0_ST_STM32F4_FLASH_CONTROLLER_LABEL
```

Comment:
The `DT_FLASH_DEV_NAME` was accidentally removed by "drivers/usb: usb_dc_stm32: Convert to DT_INST" commit a778bcd45a560d58db08da8673b360390c74095f. We need a follow up patch. Some applications, e.g. MCUBoot fail to compile.